test/test-download-manager.js

```diff
@@ -1,53 +1,228 @@
 'use strict';
 
+const del = require('del');
+const sinon = require('sinon');
+const storage = require('node-persist');
+const path = require('path');
+
+const WebDriverBrowser = require(
+  '../src/webdriver-browser/web-driver-browser.js');
+const seleniumAssistant = require('../src/index.js');
+const downloadManager = require('../src/download-manager.js');
+
 require('chai').should();
 
-describe('Test Download Manager', function() {
-  const originalPlatform = Object.getOwnPropertyDescriptor(process, 'platform');
+const testPath = './test/test-output';
+const stubs = [];
+let browserDownloads;
 
-  const downloadManager = require('../src/download-manager.js');
+describe('Test Download Manager - Browser Expiration', function() {
+  const performTest = (browserId, releases) => {
+    // 5 Minutes
+    const DOWNLOAD_TIMEOUT = 5 * 60 * 1000;
 
-  afterEach(function() {
-    Object.defineProperty(process, 'platform', originalPlatform);
-  });
+    releases.forEach(release => {
+      it(`should download ${browserId} - ${release} with no expiration.`, function() {
+        this.timeout(DOWNLOAD_TIMEOUT);
 
-  const environments = [
-    'linux',
-    'darwin',
-    'windows'
-  ];
+        return downloadManager.downloadBrowser(browserId, release)
+        .then(() => {
+          browserDownloads[browserId][release].should.equal(true);
+          browserDownloads[browserId][release] = false;
+          return downloadManager.downloadBrowser(browserId, release);
+        })
+        .then(() => {
+          // The default should be 24 hours, so manipulate the DB to say older
+          // than 24 hours
+          return new Promise((resolve, reject) => {
+            storage.initSync({
+              dir: path.join(seleniumAssistant.getBrowserInstallDir(), 'database/')
+            });
+
+            const storageKey = `${browserId}:${release}`;
+
+            const lastUpdate = Date.now() - (24 * 60 * 60 * 1000);
```

Comment:
See my earlier comment—it would be ideal to have the default of `24` exposed (e.g., as a static constant on the `DownloadManager` class) and then make use of there here, instead of hardcoding `24` in both places.